CMakeLists.txt

```diff
@@ -275,6 +275,14 @@ add_custom_target(
 add_custom_target(tests COMMENT "Building test targets")
 add_dependencies(tests hilti-tests spicy-tests)
 
+# By default Test targets are not built, but we want them to be build as part
+# of the default dev experience. Projects using Spicy can still disable them
+# when bundling Spicy by setting the the option.
+option(SPICY_ENABLE_TESTS "Build Spicy unit tests and benchmarks" ON)
```

Comment:
This comment feels misleading: by default, they *are* on, no? (I assume the comment refers to `EXCLUDE_FROM_ALL`, but still, suggest to rephrase.)

Reply:
This was unintentional, I simply confused myself. This should be fixed now.